cluster/gce/gci/configure-helper.sh

```diff
@@ -675,14 +725,22 @@ contexts:
     user: kube-apiserver
   name: webhook
 EOF
-    # This is the config for the image review admission controller.
-    cat <<EOF >/etc/admission_controller.config
+    GCP_IMAGE_REVIEW_CONFIG="imagePolicy:
```

Comment:
Is the "imagePolicy:" a surplus?

Reply:
I'm not sure but I wanted to preserve the functionality of what happens if just GCP_IMAGE_VERIFICATION_URL is set